lib/onelogin/ruby-saml/logoutresponse.rb

```diff
@@ -69,46 +104,211 @@ def status_code
         end
       end
 
+      # @return [String] Gets the StatusMessage from a SAML Response.
+      #
+      def status_message
+        @status_message ||= begin
+          node = REXML::XPath.first(document, "/p:LogoutResponse/p:Status/p:StatusMessage", { "p" => PROTOCOL, "a" => ASSERTION })
+          node.text if node
+        end
+      end
+
+      # Gets the expected current_url
+      # (Right now we read this url from the Sinle Logout Service of the Settings)
+      # TODO: Calculate the real current_url and use it.
```

Comment:
I think this method not gonna be trivial because depends on what ruby web framework be used.
We need to think in a global solution compatible with any framework and don't want to delay more this merge.